tests/migration_test.go

```diff
@@ -4451,6 +4453,72 @@ var _ = Describe("[rfe_id:393][crit:high][vendor:cnv-qe@redhat.com][level:system
 		})
 
 	})
+
+	Context("when migrating fails", func() {
+		var vmi *v1.VirtualMachineInstance
+
+		BeforeEach(func() {
+			vmi = libvmi.NewCirros(
+				libvmi.WithInterface(libvmi.InterfaceDeviceWithMasqueradeBinding()),
+				libvmi.WithNetwork(v1.DefaultPodNetwork()),
+				libvmi.WithAnnotation(v1.FuncTestForceLauncherMigrationFailureAnnotation, ""),
+			)
+		})
+
+		It("[Serial] retrying immediately should be blocked by the migration backoff", func() {
+			By("Starting the VirtualMachineInstance")
+			vmi = tests.RunVMIAndExpectLaunch(vmi, 240)
+
+			By("Waiting for the migration to fail")
+			migration := tests.NewRandomMigration(vmi.Name, vmi.Namespace)
+			_ = runMigrationAndExpectFailure(migration, tests.MigrationWaitTime)
+
+			By("Try again")
+			migration = tests.NewRandomMigration(vmi.Name, vmi.Namespace)
+			_ = runMigrationAndExpectFailure(migration, tests.MigrationWaitTime)
+
+			By("Expecting for a MigrationBackoff event to be sent")
+			eventListOpts := metav1.ListOptions{
+				FieldSelector: fmt.Sprintf("type=%s,reason=%s", k8sv1.EventTypeWarning, watch.MigrationBackoffReason),
+			}
+			expectEvents(eventListOpts, 1)
+			deleteEvents(eventListOpts)
+		})
+
+		It("[Serial] after a successful migration backoff should be cleared", func() {
```

Comment:
Why wouldn't run it in parallel?

Reply:
We are deleting events. While with some effort we can make this parallel.

Reply:
I'm just sticking to the existing functions.